x/fungible/keeper/msg_server_update_zrc20_paused_status.go

```diff
@@ -0,0 +1,65 @@
+package keeper
+
+import (
+	"context"
+
+	cosmoserrors "cosmossdk.io/errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/zeta-chain/zetacore/x/fungible/types"
+	zetaObserverTypes "github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+// UpdateZRC20PausedStatus updates the paused status of a ZRC20
+// The list of ZRC20s are either paused or unpaused
+func (k Keeper) UpdateZRC20PausedStatus(
+	goCtx context.Context,
+	msg *types.MsgUpdateZRC20PausedStatus,
+) (*types.MsgUpdateZRC20PausedStatusResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	// check message validity
+	if err := msg.ValidateBasic(); err != nil {
+		return nil, err
+	}
+
+	// check if the sender is the admin
+	if msg.Creator != k.observerKeeper.GetParams(ctx).GetAdminPolicyAccount(zetaObserverTypes.Policy_Type_deploy_fungible_coin) {
```

Comment:
Maybe lets create a new policy type ? What do you think ?

Reply:
Maybe even use `Policy_Type_stop_inbound_cctx`and rename it to `Policy_Update_Permissions`

Reply:
@kingpinXD let's consolidate the policy types. We should not have more than 2, at most 3.

Reply:
Agree with this. `Policy_Type_deploy_fungible_coin` is also used for the update bytecode message.I think `Policy_Type_deploy_fungible_coin` can be renamed into something like `Policy_Type_admin_fungible_coin`.

Reply:
Opened this one some time ago: https://github.com/zeta-chain/node/issues/932
I think we could handle it for v10